composer.json

```diff
@@ -42,7 +43,10 @@
         "twig/twig": "^2.12.1 || ^3.0"
     },
     "require-dev": {
+        "doctrine/doctrine-bundle": "^2.3.2",
+        "doctrine/orm": "^2.9",
         "friendsofphp/php-cs-fixer": "^3.4",
+        "imagine/imagine": "^1.3",
```

Comment:
Unable to fix the build without this: https://github.com/sonata-project/SonataMediaBundle/pull/2310